def divide_by(a , b):
    return a / b




try:
    ans = divide_by(54,3)
    print(ans)
except:
    print("Some thing went wrong! ")


'''
try:
    ans = print(divide_by(40,0))

except Exception as e:
    print("Some thing went wrong! ",e)
'''
  

   